### ISSUE TITLE
City siege second stage reward bug

### ISSUE THREAD

**Issue by Grinogre** [WarEmu/WarBugs#16417]
**Testing Screenshots/Videos/Evidences (always needed):**

![Grinogre_142](https://user-images.githubusercontent.com/63897509/79630512-e0943500-811f-11ea-98ac-8ef1f973d3b3.jpg)

**Comment by Kaelang** [comment]
How did Stage 2 end? Did everything function correctly?

**Comment by Grinogre** [comment]
Yes, it ended with us killing both of the order champions at middle. The
order did not kill any champions. Phases one and three gave rewards
correctly.

**Comment by Grinogre** [comment]
Also, we did still get the renown for the phase.

**Comment by Stophy22** [comment]
I was Kurodon, was about to make a bug tracker myself. We didn't get royal crests or a bag roll. Just to clarify
![Kurodon_008](https://user-images.githubusercontent.com/23489906/79678442-de31e980-81b8-11ea-9d8b-66e63667eda8.jpg)
